source/opt/aggressive_dead_code_elim_pass.h

```diff
@@ -72,7 +73,10 @@ class AggressiveDCEPass : public MemPass {
 
   // Add |inst| to worklist_ and live_insts_.
   void AddToWorklist(ir::Instruction* inst) {
-    if (live_insts_.insert(inst).second) worklist_.push(inst);
+    if (!IsLive(inst)) {
```

Comment:
This does two operations on the bit vector when before we only did one.

Seems we could use the abstraction of "ensure the bit is set, and tell me if it wasn't set before this operation".
Either that, or update Set to return a bool saying whether it actually changed the state of the value.  (I don't know if that would be slower to burden *every* Set operation that way.